docs/start/install.md

```diff
@@ -17,46 +19,46 @@ You can install the WasmEdge Runtime on any generic Linux and MacOS platforms. I
 
 ### Generic Linux and MacOS
 
-The easiest way to install WasmEdge is to run the following command. Your system should have `git` and `curl` as prerequisites.
+Your system should have `git` and `curl` as prerequisites. The easiest way to install WasmEdge is to run the following command:
```

Comment:
Could you please revert these changes as well

Reply:
Noted.